pkg/scannerv4/client/client.go

```diff
@@ -138,6 +138,7 @@ func createGRPCConn(ctx context.Context, o connOptions) (*grpc.ClientConn, error
 		clientconn.ServerName(o.serverName),
 		clientconn.MaxMsgReceiveSize(maxRespMsgSize),
 		clientconn.WithDialOptions(dialOpts...),
+		clientconn.AddRootCAs(),
```

Comment:
It doesn't seem like this should be needed. How are you running this? Completely locally or deployed via Helm?

I don't understand how this fixes the problem (I believe that it does for you, but I'm confused as to why).

If we do end up adding this, we should do it conditionally, so it doesn't affect prod deployments. From what I can tell, they've been working as-is

Reply:
> How are you running this? Completely locally or deployed via Helm?
Completely locally

> I don't understand how this fixes the problem

I explained here: https://github.com/stackrox/stackrox/pull/10687#issuecomment-2075713272
The TL;DR is `AddRootCAs` will add the CA we configure in the environment variables. I can find which paritcular ones but it's all or some subset of these:
https://github.com/stackrox/stackrox/blob/44e3d1daf95517e70468af2de6bd9d62d937e393/scanner/cmd/scannerctl/main.go#L78-L90

> If we do end up adding this, we should do it conditionally, so it doesn't affect prod deployments. From what I can tell, they've been working as-is
Yeah, I definitely not sure if this is the correct way to fix the problem. In fact, after speaking with Dave, I'm wondering why some of our TLS verification/management works the way it does in general

Reply:
I finally looked into this again, and I pretty much came to this same conclusion as you. Have you also tried running this in k8s or openshift to see if it still works in a prod environment?

Reply:
I'm actually still confused, though, as, without this, I believe we use the system's CA certs. I believe Central is typically the CA, and I don't see where we add its cert into our system's CA cert pool. Do you know where/if we do that?